src/Sylius/Component/Inventory/Checker/AvailabilityChecker.php

```diff
@@ -14,35 +14,16 @@
 use Sylius\Component\Inventory\Model\StockableInterface;
 
 /**
- * Checks availability for given stockable object.
- *
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  */
 class AvailabilityChecker implements AvailabilityCheckerInterface
 {
     /**
-     * Are backorders enabled?
-     *
-     * @var bool
-     */
-    protected $backorders;
-
-    /**
-     * Constructor.
-     *
-     * @param bool $backorders
-     */
-    public function __construct($backorders)
-    {
-        $this->backorders = (bool) $backorders;
-    }
-
-    /**
      * {@inheritdoc}
      */
     public function isStockAvailable(StockableInterface $stockable)
```

Comment:
This whole method can be replaced by `return $this->isStockSufficient($stockable, 1);`